lib/segment/src/index/field_index/map_index/mmap_map_index.rs

```diff
@@ -198,7 +207,8 @@ impl<N: MapIndexKey + Key + ?Sized> MmapMapIndex<N> {
     }
 
     pub fn get_count_for_value(&self, value: &N) -> Option<usize> {
-        match self.value_to_points.get(value) {
+        let hw_counter = HardwareCounterCell::disposable(); // TODO(io_measurement): Maybe propagate?
```

Comment:
Yeah I think we need to count this, IIRC its used in the facet API and it can be expensive.